packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/test/feature_counter/views/counter_page_test.dart

```diff
@@ -0,0 +1,48 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:provider/provider.dart';
+
+import 'package:{{project_name}}/feature_counter/blocs/counter_bloc.dart';
+import 'package:{{project_name}}/feature_counter/views/counter_page.dart';
+
+import '../../helpers/golden_helper.dart';
+import '../../helpers/models/scenario.dart';
+import '../mocks/counter_bloc_mock.dart';
+
+Widget counterPageFactory({
+  String? error,
+  int? count,
+  bool? isLoading,
+}) =>
+    Provider<CounterBlocType>(
+      create: (_) => counterBlocMockFactory(
+        count: count,
+        error: error,
+        isLoading: isLoading,
+      ),
+      child: const CounterPage(),
+    );
+
+void main() {
+  group('CounterPage golden tests', () {
+    final deviceBuilders = [
+      generateDeviceBuilder(
+        label: 'counter',
+        widget: counterPageFactory(count: 2),
+        scenarios: [Scenario(name: 'Default')],
+      ),
+      generateDeviceBuilder(
+        label: 'error',
+        widget: counterPageFactory(count: 2, error: 'Test errors'),
+        scenarios: [Scenario(name: 'Error scenario')],
```

Comment:
What is the benefit of providing a list of Scenarios? From what I see we test just one scenario. Could you please elaborate?

Reply:
This was left over from a previous implementation, corrected